seed/tests/test_portfoliomanager.py

```diff
@@ -365,3 +365,66 @@ def test_report_generation_empty_child_template(self):
         # in this case, we expect a meaningful error message
         self.assertIn('message', body)
         self.assertIn('empty', body['message'])
+
+
+class PortfolioManagerReportSinglePropertyUploadTest(TestCase):
+
+    def setUp(self):
+        user_details = {
+            'username': 'test_user@demo.com',
+            'password': 'test_pass',
+        }
+        self.user = User.objects.create_superuser(
+            email='test_user@demo.com', **user_details
+        )
+        self.org, _, _ = create_organization(self.user)
+        self.client.login(**user_details)
+
+        # create a dataset
+        dataset_name = 'test_dataset'
+        response = self.client.post(
+            reverse_lazy('api:v3:datasets-list') + '?organization_id=' + str(self.org.pk),
+            data=json.dumps({'name': dataset_name}),
+            content_type='application/json',
+        )
+        dataset = response.json()
+        self.dataset_id = dataset['id']
+
+        self.pm_un = os.environ.get(PM_UN, False)
+        self.pm_pw = os.environ.get(PM_PW, False)
+        if not self.pm_un or not self.pm_pw:
+            self.fail('Somehow PM test was initiated without %s or %s in the environment' % (PM_UN, PM_PW))
+
+    def test_single_property_template_for_upload(self):
```

Comment:
I think we need to add the `pm_skip_test_check` decorator here like the other PM tests to make sure it's skipped if we're missing the env vars or if PM is unavailable